jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I would not use the word "extreme" or "great" - perhaps something more specific like "distinct transcriptional differences" or "large magnitude of transcriptional change"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>